homeassistant/components/netatmo/strings.json

```diff
@@ -53,7 +53,9 @@
     "trigger_subtype": {
       "away": "[%key:common::state::not_home%]",
       "schedule": "Schedule",
-      "hg": "Frost guard"
+      "hg": "Frost guard",
+      "temperature": "Temperature",
```

Comment:
use the common translation key here

Reply:
and what would that be? I found nothing for temperature inside `strings.json`

Reply:
`[%key:component::sensor::entity_component::temperature::name%]`

Reply:
`[%key:component::sensor::entity_component::temperature::name%]`